Compiling Javascript with lineman failed

I'm trying to remove the need for installing `lineman` globally, that way there is no need for running `npm install -g lineman` to been able to use `lineman` inside the `frontend` folder. Since is already declared as a dependency in our `package.json`, we can make use of the scripts section.

``` javascript
{
  "name": "frontend",
  "title": "An HTML/JS/CSS app",
  "version": "0.0.1",
  "private": true,
  "author": {
    "name": "John Doe",
    "company": "ACME, Inc."
  },
  "scripts": {
    "watch": "lineman run",
    "build:production": "lineman build"
  },
  "dependencies": {
    "grunt": "~0.4.1",
    "grunt-ngmin": "~0.0.2",
    "lineman": ">=0.11.1"
  },
  "engines": {
    "node": "~>0.10.5",
    "npm": "~>1.2.18"
  }
}
```

by running `npm run watch` or `npm run build:production` will execute the line specify in the scripts section.

Without the need of installing lineman globally I'm able to run `npm run watch` and lineman will watch for javascripts files, but trying to run `npm run build:production` I get this error: 

``` js
> frontend@0.0.1 build:production /Users/morty/dev/exercism_project/exercism.io/frontend
> lineman build

Running "clean:dist" (clean) task
Cleaning generated...OK

Running "common" task

Running "coffee:compile" (coffee) task
File generated/js/app.coffee.js created.
File generated/js/spec.coffee.js created.
>> Destination (generated/js/spec-helpers.coffee.js) not written because compiled files were empty.

Running "jshint:files" (jshint) task
Warning: Path must be a string. Received null Use --force to continue.

Aborted due to warnings.

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/morty/.nvm/versions/node/v6.3.1/bin/node" "/Users/morty/.nvm/versions/node/v6.3.1/bin/npm" "run" "build:production"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! frontend@0.0.1 build:production: `lineman build`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the frontend@0.0.1 build:production script 'lineman build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the frontend package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     lineman build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs frontend
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls frontend
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/morty/dev/exercism_project/exercism.io/frontend/npm-debug.log
```

So I decided to installed globally to see if it's failing as well I typed `npm install -g lineman` then went back to the `frontend` folder I try `lineman build` will fail again with:

``` js
Running "clean:dist" (clean) task
Cleaning generated...OK

Running "common" task

Running "coffee:compile" (coffee) task
File generated/js/app.coffee.js created.
File generated/js/spec.coffee.js created.
>> Destination (generated/js/spec-helpers.coffee.js) not written because compiled files were empty.

Running "jshint:files" (jshint) task
Warning: Path must be a string. Received null Use --force to continue.

Aborted due to warnings.
```

I don't know if someone else is experience this kind of errors, but maybe we have revise the building process for javascript assets.

By the way I use `nvm` for managing my nodes versions. Maybe that has something to do.

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "grunt": "~0.4.1",
     "grunt-concat-sourcemap": "^0.4.3",
     "grunt-ngmin": "~0.0.2",
-    "lineman": ">=0.11.1"
+    "lineman": "~0.37.0"
   },
   "engines": {
     "node": "~>0.10.5",

